Packet++/src/GtpLayer.cpp

```diff
@@ -155,522 +155,521 @@ GtpV1Layer::GtpExtension GtpV1Layer::GtpExtension::createGtpExtension(uint8_t* d
+bool GtpV1Layer::isGTPv1(const uint8_t *data, size_t dataSize)
 {
-    if (data != NULL && dataSize > 1 && (data[0] & 0xE0) == 0x20)
-    {
-        return true;
-    }
+	if(data != NULL && dataSize > 1 && (data[0] & 0xE0) == 0x20)
+	{
+		return true;
+	}
 
-    return false;
+	return false;
 }
 
-GtpV1Layer::gtpv1_header_extra* GtpV1Layer::getHeaderExtra()
+GtpV1Layer::gtpv1_header_extra *GtpV1Layer::getHeaderExtra() const
 {
-    if (m_Data != NULL && m_DataLen >= sizeof(gtpv1_header) + sizeof(gtpv1_header_extra))
-    {
-        return (gtpv1_header_extra*)(m_Data + sizeof(gtpv1_header));
-    }
+	if (m_Data != NULL && m_DataLen >= sizeof(gtpv1_header) + sizeof(gtpv1_header_extra))
+	{
+		return (gtpv1_header_extra *)(m_Data + sizeof(gtpv1_header));
+	}
 
-    return NULL;
+	return NULL;
 }
 
-bool GtpV1Layer::getSequenceNumber(uint16_t& seqNumber)
+bool GtpV1Layer::getSequenceNumber(uint16_t &seqNumber) const
 {
-    gtpv1_header* header = getHeader();
-    gtpv1_header_extra* headerExtra = getHeaderExtra();
-    if (header != NULL && headerExtra != NULL && header->sequenceNumberFlag == 1)
-    {
-        seqNumber = be16toh(headerExtra->sequenceNumber);
-        return true;
-    }
+	gtpv1_header *header = getHeader();
+	gtpv1_header_extra *headerExtra = getHeaderExtra();
+	if (header != NULL && headerExtra != NULL && header->sequenceNumberFlag == 1)
+	{
+		seqNumber = be16toh(headerExtra->sequenceNumber);
+		return true;
+	}
 
-    return false;
+	return false;
 }
 
 bool GtpV1Layer::setSequenceNumber(const uint16_t seqNumber)
 {
-    // get GTP header
-    gtpv1_header* header = getHeader();
-    if (header == NULL)
-    {
-        LOG_ERROR("Set sequence failed: GTP header is NULL");
-        return false;
-    }
+	// get GTP header
+	gtpv1_header *header = getHeader();
+	if (header == NULL)
+	{
+		LOG_ERROR("Set sequence failed: GTP header is NULL");
+		return false;
+	}
 
-     // if all flags are unset then create the GTP extra header
-    if (header->npduNumberFlag == 0 && header->sequenceNumberFlag == 0 && header->extensionHeaderFlag == 0)
-    {
-        if (!extendLayer(sizeof(gtpv1_header), sizeof(gtpv1_header_extra)))
-        {
-            LOG_ERROR("Set sequence failed: cannot extend layer");
-            return false;
-        }
-        header = getHeader();
-    }
+	// if all flags are unset then create the GTP extra header
+	if (header->npduNumberFlag == 0 && header->sequenceNumberFlag == 0 && header->extensionHeaderFlag == 0)
+	{
+		if(!extendLayer(sizeof(gtpv1_header), sizeof(gtpv1_header_extra)))
+		{
+			LOG_ERROR("Set sequence failed: cannot extend layer");
+			return false;
+		}
+		header = getHeader();
+	}
 
-    // get the extra header
-    gtpv1_header_extra* headerExtra = getHeaderExtra();
-    if (headerExtra == NULL)
-    {
-        LOG_ERROR("Set sequence failed: extra header is NULL");
-        return false;
-    }
+	// get the extra header
+	gtpv1_header_extra *headerExtra = getHeaderExtra();
```

Comment:
please move the `*` back to its original place. This is the convention in the rest of the project